src/models/position/facade.js

```diff
@@ -162,12 +162,16 @@ const prepareChangeParticipantPositionTransaction = async (transferList) => {
         const participantLimit = await participantFacade.getParticipantLimitByParticipantCurrencyLimit(participantCurrency.participantId, participantCurrency.currencyId, Enum.Accounts.LedgerAccountType.POSITION, Enum.Accounts.ParticipantLimitType.NET_DEBIT_CAP)
         // Calculate liquidity cover as per story OTC-651
         let liquidityCover
+        let payerLimit
         if (settlementModel.settlementDelayId === Enum.Settlements.SettlementDelay.IMMEDIATE) {
           liquidityCover = new MLNumber(settlementParticipantPosition.value).add(new MLNumber(participantLimit.value))
+          payerLimit = new MLNumber(participantLimit.value)
         } else {
-          liquidityCover = new MLNumber(participantLimit.value)
+          liquidityCover = new MLNumber(settlementParticipantPosition.value).multiply(-1)
```

Comment:
Just to confirm, Is this to make the amount positive?

Reply:
Yes, that's right